src/plugins/profiling/README.md

```diff
@@ -4,6 +4,61 @@ A Kibana plugin
 
 ---
 
+## Getting started
+
+### Prerequisites
+
+1. nvm: See [installation instructions](https://github.com/nvm-sh/nvm#install--update-script)
+1. Elasticsearch: See [Local development environment setup](https://docs.google.com/document/d/1gzK8yC_MptfMJkx6QAud6GSv8br0WcYW4vuX0VeuNrI/edit) to start an Elasticsearch cluster
+
+### Configure local Kibana environment
+
+The easiest way to configure your local Kibana environment is to create a
+`config/kibana.dev.yml` in the repo's root directory.
+
+Example configuration:
+
+```yaml
+# https://www.elastic.co/guide/en/kibana/8.0/settings.html
+
+# If you run the Kibana dev environment within a container or wish to access the
+# UI beyond your local machine, you will need this setting
+
+server.host: "0.0.0.0"
+
+# If there is a mismatch between your local Kibana env and the running Elasticsearch
+# cluster, then this setting will change the error message to a warning
+
+elasticsearch.ignoreVersionMismatch: true
+
+# Required to connect to your Elasticsearch cluster
+# (settings below assume you are using elastic/apm-integration-testing)
+# Adjust as necessary
+
+elasticsearch.hosts: "http://localtesting_8.0.0_elasticsearch:9200"
```

Comment:
Not sure about the name resolution here... for me `localhost:9200` works best when ES is started with apm-integrtion-testing

Reply:
Good point, I've updated the `hosts` config.